src/cromshell/update_server/command.py

```diff
@@ -0,0 +1,41 @@
+import json
+import logging
+import os
+from pathlib import Path
+
+import click
+
+LOGGER = logging.getLogger(__name__)
+CROMWELL_SERVER_KEY = "cromwell_server"
+
+
+@click.command(name="update-server")
+@click.argument("cromwell_server_url", required=True, nargs=1)
+@click.pass_obj
+def main(config, cromwell_server_url):
+    """Update the default Cromwell server in the config file"""
+
+    # location of the config file
+    cromshell_config_path = os.path.join(
+        config.config_dir, config.CROMSHELL_CONFIG_FILE_NAME
+    )
+    assert os.access(
+        cromshell_config_path, mode=os.W_OK
+    ), f"Cannot write to Cromshell config file {cromshell_config_path}"
+
+    # the contents of the config file as a dict
+    cromshell_config_dict = config.cromshell_config_options
+
+    # update the cromwell server
+    if CROMWELL_SERVER_KEY not in cromshell_config_dict.keys():
+        cromshell_config_dict.update({CROMWELL_SERVER_KEY: cromwell_server_url})
+    else:
+        cromshell_config_dict[CROMWELL_SERVER_KEY] = cromwell_server_url
+
+        # write the modified config file
```

Comment:
Would this part need to be tab-shifted to the left so that it updates the cromshell config file all the time, and not just when it finds a cromwell_server_key in the cromshell_config_options?

Reply:
You are totally right, I just saw that last night and converted this back to a draft PR so I wouldn't forget to fix that, good catch.

Reply:
It "shouldn't" be missing a `cromwell_server`. On the initial run, cromshell will look for the config json and if it doesn't find it, it will create one from a template that has the `cromwell_server` key. Then it will ask the user to update its value. It's possible the user could create their own or edit a config file themselves and remove the key, this would cause cromshell to fail since it won't be able to access the Cromwell server. In the long run, I'd like to add some validation scripts for the config file to make sure certain keys are available.